DNN Platform/Library/Entities/Tabs/TabVersions/TabVersionController.cs

```diff
@@ -112,6 +119,15 @@ private static string GetTabVersionsCacheKey(int tabId)
         {
             return string.Format(DataCache.TabVersionsCacheKey, tabId);
         }
+
+        private IEnumerable<TabVersion> ApplyUserLocalTime(IEnumerable<TabVersion> tabVersions, TimeSpan userUtcOffset, TimeZoneInfo userTimeZone)
+        {
+            return tabVersions.Select((tabVersion) =>
+            {
+                tabVersion.CreateOnUserLocalDate = System.TimeZoneInfo.ConvertTime(tabVersion.CreatedOnDate, userTimeZone);
```

Comment:
Because `CreatedOnDate` is a `DateTime` the conversion can produce incorrect result (please see *Remarks* section on [this page](https://msdn.microsoft.com/en-us/library/bb382835(v=vs.110).aspx).

Hence my earlier question. What is the representation of the times in the database? They are server time, right? But what is their `DateTimeKind`?

Specifically, if it's `Local` then the source timezone should be retrieved and passed explicitly to the [`TimeZoneInfo.ConvertTime Method (DateTime, TimeZoneInfo, TimeZoneInfo)`](https://msdn.microsoft.com/en-us/library/bb382770(v=vs.110).aspx) overload